content/en/docs/pluggable-components/events.md

```diff
@@ -47,7 +47,7 @@ By default, KubeKey will install Elasticsearch internally if Events is enabled.
       elasticsearchDataVolumeSize: 20Gi    # The volume size of Elasticsearch data nodes.
       logMaxAge: 7                     # Log retention day in built-in Elasticsearch. It is 7 days by default.
       elkPrefix: logstash              # The string making up index names. The index name will be formatted as ks-<elk_prefix>-log.
-      externalElasticsearchHost: # The Host of external Elasticsearch.
+      externalElasticsearchUrl: # The URL of external Elasticsearch.
```

Comment:
`The URL of external Elasticsearch` => `The Host of external Elasticsearch`

`Host` is more accurate, please also modify other places.

Reply:
Thanks and updated in [this commit](https://github.com/kubesphere/website/pull/2289/commits/3869e9d2b36c6ca22851c11ee99c5a4dbf3a6e28).